### PR TITLE
[react-spinkit] Fill out props and add testcases

### PR DESCRIPTION
**Pull request by pelotom**
- Fill out missing props based on https://github.com/KyleAMathews/react-spinkit#proptypes
- Add testcases based on examples from http://kyleamathews.github.io/react-spinkit/

### CODE REVIEW

**Comment by tkqubo** [comment]
👍

File: react-spinkit/react-spinkit-tests.tsx [hunk 1]
```diff
@@ -4,4 +4,23 @@
...
+    <Spinner spinnerName="wave" />,
+    <Spinner spinnerName="wandering-cubes" />,
+    <Spinner spinnerName="pulse" />,
+    <Spinner spinnerName="chasing-dots" />,
+    <Spinner spinnerName="circle" />,
+    <Spinner spinnerName="cube-grid" />,
+    <Spinner spinnerName="wordpress" />,
+
+    // Spinner options
+    < Spinner spinnerName= "wordpress" noFadeIn />,
```
Comment (mleko) [inline react-spinkit/react-spinkit-tests.tsx]: Remove extra space:)

**Comment by mleko** [comment]
👍

File: react-spinkit/react-spinkit-tests.tsx [hunk 2]
```diff
@@ -4,4 +4,23 @@
...
+    <Spinner spinnerName="wave" />,
+    <Spinner spinnerName="wandering-cubes" />,
+    <Spinner spinnerName="pulse" />,
+    <Spinner spinnerName="chasing-dots" />,
+    <Spinner spinnerName="circle" />,
+    <Spinner spinnerName="cube-grid" />,
+    <Spinner spinnerName="wordpress" />,
+
+    // Spinner options
+    < Spinner spinnerName="wordpress" noFadeIn />,
```
Comment (mleko) [inline react-spinkit/react-spinkit-tests.tsx]: One more
```
< Spinner
 ^
```

**Pull request merged**